inc/import-users.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+function hale_add_import_users_page() {
+
+    if (current_user_can('manage_options')) {
+        add_menu_page(
+            'Import Users',
+            'Import Users',
+            'manage_options',
+            'import-users',
+            'hale_import_users_page',
+            'dashicons-admin-generic'
+        );
+    }
+}
+
+add_action('admin_menu', 'hale_add_import_users_page');
+
+function hale_import_users_page() {
+    // Check if the current user has the 'manage_options' capability (administrator)
+    if (!current_user_can('manage_options')) {
+        wp_die(__('You do not have sufficient permissions to access this page.'));
+    }
+
+    // Define variables to store form data and error messages
+    $json_data = '';
+    $errors = [];
+    $added_users = 0;
+    $failed_users = 0;
+
+    // Check if the form is submitted
+    if (isset($_POST['submit'])) {
+        // Retrieve form data
+        $json_data = stripslashes($_POST['json_data']);
```

Comment:
I guess it is data we are adding but is it worth adding in sanitize_text_field() so `$json_data = sanitize_text_field(stripslashes($_POST['json_data']));
`?

Reply:
orignally had that but it then failed to parse as JSON